product_docs/docs/tde/15/key_stores.mdx

```diff
@@ -61,16 +63,17 @@ postgres ALL = NOPASSWD: /usr/bin/systemd-ask-password
 You can use the key store in an external key management system to manage the data encryption key. The tested and supported key stores are:
 
 - Amazon AWS Key Management Service (KMS)
+- Microsoft Azure Key Vault
 - Google Cloud - Cloud Key Management Service
 - HashiCorp Vault (KMIP Secrets Engine and Transit Secrets Engine)
-- Microsoft Azure Key Vault
 - Thales CipherTrust Manager
```

Comment:
@theadamwright  I noticed we have examples for all key stores except for Thales CipherTrust. I added more information about my findings in https://enterprisedb.atlassian.net/browse/DOCS-1038?focusedCommentId=639364 but I think it would be good to add an example here as well.